DEPENDENCIES.rst

```diff
@@ -165,6 +165,11 @@ To determine which version of Python is already installed on your computer, run:
 
    python -V
 
+
+Cython Note
+^^^^^^^^^^
+If you get any error related to cython version then this maybe because debian stable version of cython is .23.4 but cyclus installation requires .25+. To get latest cython version installed please visit the `Cython Documentation`_.
```

Comment:
This line is too long, and will need to be line-wrapped so that it is many shorter lines. Let's go with a little rephrasing as well, just for grammar:

```
If you get an error related to an old Cython version, then this may be 
because the Debian stable version of Cython is 0.23.4. However, 
Cyclus requires .25+. To install the latest cython version, please 
visit the `Cython Documentation`_.
```